specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/DotNetComponents.json

```diff
@@ -405,8 +405,7 @@
             "componentType": {
               "description": "Type of the .NET Component.",
               "enum": [
-                "AspireDashboard",
-                "AspireResourceServerApi"
+                "AspireDashboard"
```

Comment:
Also, need to update same for `2023-11-02-preview`

Reply:
Thanks, here it is: https://github.com/Azure/azure-rest-api-specs/pull/28096/files